CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Cromwell Change Log
 
+## 82 Release Notes
+
+ * Restored missing example configuration file
+ * Upgraded to latest version of googleCloudNioV (0.124.8)
```

Comment:
I absolutely am responsible for the proliferation of this after having it in my PR title, but `googleCloudNioV` is our own internal variable name while the library is really called Google Cloud Storage NIO

```suggestion
 * Upgraded to latest version of the Google Cloud Storage NIO library (0.124.8)
```

Reply:
Thank you!